domains/README.md

```diff
@@ -16,6 +16,29 @@ NOTE: currently, the domain chain does not support to syncing from other operato
 
 The domain operator node is embeded within the `subspace-node` binary, please refer to [Subspace node](../crates/subspace-node/README.md) for how to build from source.
 
+#### Create Operator key:
+Operator needs key pair to participate in Bundle production.
+You can create a key using following command:
+```bash
+target/production/subspace-node key generate --scheme sr25519
+```
+
+Backup the key. Take `Public key (hex)` of the Keypair. The public key is part of the Operator config.
+
+#### Insert key to Keystore:
+The above generated key is added to the Keystore so that Operator node can use to participate in Bundle production.
+The key is inserted using the following command:
+```bash
+target/production/subspace-node key insert \
+      --suri "<Secret phrase>" --key-type <key1> --scheme sr25519 --keystore-path /tmp/keystore
+```
+The above command assumes `/tmp/keystore` as the keystore location. `key-type` should a 4-letter word.
```

Comment:
Shouldn't it be a very specific 4-letter word? I thought it'd be something pre-defined like `sub_` that we have on consensus side.

Reply:
True, its `oper` actually